administration/src/cards/createCards.ts

```diff
@@ -22,7 +22,7 @@ export class CreateCardsError extends Error {
   }
 }
 
-async function createCards(client: ApolloClient<object>, activationCodes: Codes, region: Region) {
+async function createCards(client: ApolloClient<object>, project: string, activationCodes: Codes, region: Region) {
```

Comment:
@f1sh1918  I realized that I added the project here. Is it okay to change endpoints in the administration, as we ship changes in web directly this might only mess with caching I guess. The other option would be to omit adding the project and instead extract it from the region. Any opinion on this?

Reply:
Should be okay in this case.

Reply:
as long as you don't use the endpoint in the flutter app thats fine. we ship backend and administration together